### DISCUSSION TITLE
shared library support

### DISCUSSION THREAD

**Discussion by fver** [icloud-photos-downloader/icloud_photos_downloader#695] [General]
It looks like the shared library can only be accessed by the account that owns it in 1.16.1.
Can anybody confirm this?
Apart from the shared library albums (that you also cannot access in photos itself), It should be possible to access the library from the receiving accounts.

**Comment by CreativePR** [comment]
Tried this on an owned account and did --list-albums and a shared album created and owned by the owner does not show in the list of available albums that can be downloaded.
